ShapeScript/Interpreter.swift

```diff
@@ -1277,6 +1277,8 @@ extension Expression {
                 return .number(lhs.doubleValue * rhs.doubleValue)
             case .divide:
                 return .number(lhs.doubleValue / rhs.doubleValue)
+            case .modulo:
+                return .number(lhs.doubleValue % rhs.doubleValue)
```

Comment:
Build is failing here with the warning:

`'%' is unavailable: For floating point numbers use truncatingRemainder instead`

Basically, `%` in Swift is only supported for integer operands, and every numeric value in ShapeScript is actually a double precision float. Using the `truncatingRemainder()` method as suggested seems like the right solution, but it will be worth writing some tests involving floating point and negative values to verify they behave the way you expect: https://developer.apple.com/documentation/swift/double/truncatingremainder(dividingby:)

(We'll have to implement the parser first though)